src/library/trackmodeliterator.h

```diff
@@ -0,0 +1,72 @@
+/// Utilities for iterating through a selection or collection
+/// of tracks.
+
+#pragma once
+
+#include <QModelIndex>
+
+#include "track/trackiterator.h"
+
+class TrackModel;
+
+namespace mixxx {
+
+/// Iterate over selected and valid(!) track ids in a TrackModel.
```

Comment:
```suggestion
/// Iterate over selected and valid track ids in a TrackModel.
```
I presume that `(!)` is trying to warn me about something but it doesn't explain what that is... is it just that invalid track IDs are skipped silently as explained on the next line? If so, then that's already explained below.